apps/extension/src/ui/domains/Swap/hooks/useSimpleswapSwapStatus.ts

```diff
@@ -0,0 +1,36 @@
+import { atom, useAtom, useAtomValue } from "jotai"
+import { atomFamily, unwrap } from "jotai/utils"
+import { useEffect } from "react"
+
+import {
+  completedSwapsCacheAtom,
+  saveCompletedSwapsCache,
+  simpleswapSwapStatusAtom,
+} from "../swap-modules/simpleswap-swap-module"
+
+export const useSimpleswapSwapStatus = (id?: string) => {
```

Comment:
This could be a lot simpler, but jotai was giving me a headache in `setState called during render`-esque infinite react rerenders. Keen to `rm` it when I move all of this into rxjs.

Reply:
The infinite rerenders happened when I tried to make an atom which combined the cache and the live fetch atoms. Ended up merging them inside this hook instead.